### PR TITLE
First pass at Chemaxon content

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/05-Using.mdx [hunk 1]
```diff
@@ -124,15 +119,17 @@ Indexes:
...
 
-Substructure search finds all structures that contain the query structure as a subgraph. Sometimes the chemical subgraph is not the only thing that is provided, but certain query features that further restrict the structure are also provided as well. If special molecular features are present on the query (eg. stereochemistry, charge, etc.), then only those targets match the ones which also contain the feature. However, if a feature is missing from the query, it is not required to be missing (by default). 
+### Substructure search
 
-Substructure search is performed using the symmetrical sub-/super-structure search operator: `|<|`.
+Substructure search finds all structures that contain the query structure as a subgraph. Sometimes the chemical subgraph isn't the only thing that's provided. Certain query features that further restrict the structure are also provided. If special molecular features are present on the query, such as stereochemistry or charge, then only those targets match the ones that also contain the feature. However, if a feature is missing from the query, it doesn't need to be missing, by default.
```
Comment (ebgitelman) [inline advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/05-Using.mdx:129]: Please double check these last two sentences. They were hard for me to understand.

File: advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 directoryDefaults:
     iconName: handshake
 ---
 
-<p align="center">
-  <img  src="Images/PartnerProgram.png">
-</p>
+![EDB Partner Program logo](images/PartnerProgram.png)
 <h1 style="text-align: center;">EDB GlobalConnect Technology Partner Implementation Guide</h1>
 <h3 style="text-align: center;">Chemaxon JChem PostgreSQL Cartridge</h3>
```
Comment (ebgitelman) [inline advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx:10]: @drothery-edb I wasn't sure what to do with the titles here. They are in HTML and look very much like book titles, but I wasn't sure whether I should delete them or use part of it in the description.

Comment (drothery-edb) [reply advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx:10]: We decided to look at possible improvements to the landing page format separately.

File: advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/03-SolutionSummary.mdx [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 
-The Chemaxon JChem PostgreSQL Cartridge integrates chemical structure handling with EDB Postgres Advanced Server (EPAS) and PostgreSQL through extensions to PostgreSQL's native SQL language. It enables scientists to create, manage, and analyze chemical structures and related non-chemical data in their Postgres database. JChem users can search data by Superstructure, Substructure, Duplicate and Full fragment (exact fragment) search.
+The Chemaxon JChem PostgreSQL Cartridge integrates chemical structure handling with EDB Postgres Advanced Server and PostgreSQL through extensions to PostgreSQL's native SQL language. It enables scientists to create, manage, and analyze chemical structures and related nonchemical data in their Postgres database. JChem users can search data by superstructure, substructure, duplicate, and full-fragment (exact fragment) search.
 
+![Chemaxon solutions summary](images/UpdatedChemaxonSolutionSummary.png)
```
Comment (dwicinas) [inline advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/03-SolutionSummary.mdx:10]:
```suggestion
![Chemaxon solutions summary](Images/UpdatedChemaxonSolutionSummary.png)
```

**Review by dwicinas — DISMISSED** [review]

File: advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx [hunk 4]
```diff
@@ -5,10 +5,8 @@ directoryDefaults:
     iconName: handshake
 ---
 
-<p align="center">
-  <img  src="Images/PartnerProgram.png">
-</p>
+![EDB Partner Program logo](images/PartnerProgram.png)
```
Comment (dwicinas) [inline advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx:8]:
```suggestion
![EDB Partner Program logo](Images/PartnerProgram.png)
```

**Review by dwicinas — APPROVED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**